### DISCUSSION TITLE
How to turn on/off the embla carousel by breakpoint in React?

### DISCUSSION THREAD

**Discussion by bmarshall511** [davidjerleke/embla-carousel#304] [Help] [answered]
I'm trying to turn/on off the carousel based on breakpoint. So for mobile, it's on and for tablet up it's off. Here's what I tried that seems to work initially, but won't reinitialize. I'm guessing it's because ```destory``` was called so it can ```reInit``` and tried init, but no luck there either.

```
const emblaOptions = {};
  const [viewportRef, emblaApi] = useEmblaCarousel(emblaOptions);

  const handleEmblaInit = () => {
    if (window.innerWidth > 768) {
      emblaApi.destroy();
    } else {
      emblaApi.reInit(emblaOptions);
    }
  };

  useEffect(() => {
    if (emblaApi) {
      handleEmblaInit();
      window.addEventListener("resize", handleEmblaInit);
    }
  }, [emblaApi]);
```

**Comment by davidjerleke** [comment, marked as answer]
Hi @bmarshall511,

Thank you for your question. Pass `null` instead of the `emblaRef` when you want it to be inactive, like [demonstrated here](https://github.com/davidcetinkaya/embla-carousel/issues/99#issuecomment-688730519).

- Comment from #99 

Best,
David

**Reply by bmarshall511** [reply]
Perfect, thanks!

**Reply by davidjerleke** [reply]
Hi @bmarshall511,

Just wanted to let you know that implementing this is now much easier using the [active](https://www.embla-carousel.com/api/options/#active) option together with the [breakpoints](https://www.embla-carousel.com/api/options/#breakpoints) option.

See the following [example](https://www.embla-carousel.com/guides/breakpoints/#changing-options).

Cheers,
David
